web/ui/react-app/src/pages/Config.tsx

```diff
@@ -1,47 +1,53 @@
-import React, { FC, useState } from 'react';
+import React, { useState, FC } from 'react';
 import { RouteComponentProps } from '@reach/router';
-import { Alert, Button } from 'reactstrap';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { Button } from 'reactstrap';
 import CopyToClipboard from 'react-copy-to-clipboard';
-import { useFetch } from '../utils/useFetch';
 import PathPrefixProps from '../PathPrefixProps';
 
 import './Config.css';
+import { withStatusIndicator } from '../withStatusIndicator';
+import { useFetch } from '../utils/useFetch';
 
-const Config: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => {
-  const { response, error } = useFetch(`${pathPrefix}/api/v1/status/config`);
-  const [copied, setCopied] = useState(false);
+type YamlConfig = { yaml: string };
+
+interface ConfigContentProps {
+  error?: Error;
+  data?: YamlConfig;
+}
+
+const YamlContent = ({ yaml }: YamlConfig) => <pre className="config-yaml">{yaml}</pre>;
+YamlContent.displayName = 'Config';
 
-  const config = response && response.data.yaml;
+const ConfigWithResponseIndicator = withStatusIndicator(YamlContent);
```

Comment:
I'd call it `ConfigWithStatusIndicator` for consistency and because it's clearer. Same for the other pages.